src/System.Management.Automation/engine/Modules/ModuleUtils.cs

```diff
@@ -16,35 +16,31 @@ internal static class ModuleUtils
         //  - Ignore files/directories when access is denied;
         //  - Search top directory only.
         private static readonly System.IO.EnumerationOptions s_defaultEnumerationOptions =
-                                        new System.IO.EnumerationOptions() { AttributesToSkip = 0 };
+                                        new System.IO.EnumerationOptions() { AttributesToSkip = System.IO.FileAttributes.Hidden };
 
         // Default option for UNC path enumeration. Same as above plus a large buffer size.
         // For network shares, a large buffer may result in better performance as more results can be batched over the wire.
         // The buffer size 16K is recommended in the comment of the 'BufferSize' property:
         //    "A "large" buffer, for example, would be 16K. Typical is 4K."
         private static readonly System.IO.EnumerationOptions s_uncPathEnumerationOptions =
-                                        new System.IO.EnumerationOptions() { AttributesToSkip = 0, BufferSize = 16384 };
+                                        new System.IO.EnumerationOptions() { AttributesToSkip = System.IO.FileAttributes.Hidden, BufferSize = 16384 };
 
         /// <summary>
-        /// Check if a directory could be a module folder.
+        /// Check if a directory is likely a localized resources folder.
         /// </summary>
-        internal static bool IsPossibleModuleDirectory(string dir)
+        /// <param name="dir">Directory to check if it is a possible resource folder.</param>
+        /// <returns>True if the directory name matches a culture.</returns>
+        internal static bool IsPossibleResourceDirectory(string dir)
         {
-            // We shouldn't be searching in hidden directories.
-            FileAttributes attributes = File.GetAttributes(dir);
-            if (0 != (attributes & FileAttributes.Hidden))
-            {
-                return false;
-            }
-
             // Assume locale directories do not contain modules.
-            if (dir.EndsWith(@"\en", StringComparison.OrdinalIgnoreCase) ||
-                dir.EndsWith(@"\en-us", StringComparison.OrdinalIgnoreCase))
+            if (dir.EndsWith(Path.DirectorySeparatorChar + "en", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
These are likely to allocate on every comparison -- maybe worth elevating into readonly statics?

Reply:
Will update